packages/airnode-abi/src/decoding.ts

```diff
@@ -0,0 +1,74 @@
+import { ethers } from 'ethers';
+import chunk from 'lodash/chunk';
+import { PARAMETER_SHORT_TYPES } from './utils';
+import { ABIParameterType, DecodedMap } from './types';
+
+type TransformationFunction = (value: any) => string;
+
+type TransformationReference = {
+  [key in ABIParameterType]: TransformationFunction | null;
+};
+
+// Certain types need to be parsed after ABI decoding happens
+const TRANSFORMATIONS: TransformationReference = {
+  bytes: ethers.utils.parseBytes32String,
```

Comment:
If ethers decodes a `bytes` type as `"0xabc123"`, we want that exact value and not have it parsed as a string. So this should also be null.